web/src/scss/content/panel/_time-slider.scss

```diff
@@ -40,6 +41,7 @@
     padding: 0;
     overflow: visible;
     border: none;
+    margin: 5px 0;
```

Comment:
5px is very much a magic variable... This makes the vertical margin look the same as before. 🤔